src/Main/Main.css

```diff
@@ -0,0 +1,41 @@
+ .main {
+   background: #282D35;
+   width: 100vw;
+   height: 100vh;
+   margin: auto;
+   display: inline-block;
```

Comment:
- [ ] Use Flex display here
- [ ] use `%` for the width
- [ ] Remove the margin